src/bci_build/package/__init__.py

```diff
@@ -146,35 +150,80 @@ def parse(val: str) -> OsVersion:
     def __str__(self) -> str:
         return str(self.value)
 
+    @property
+    def is_sles(self) -> bool:
+        try:
+            int(self.value)
+            return True
+        except ValueError:
+            return False
+
+    @property
+    def is_slcc(self) -> bool:
+        return self.value in (
+            OsVersion.SLCC_DEVELOPMENT.value,
+            OsVersion.SLCC_LTS.value,
+            OsVersion.SLCC_PRODUCTION.value,
+        )
+
+    @property
+    def is_opensuse(self) -> bool:
+        return self.value == OsVersion.TUMBLEWEED.value
+
+    @property
+    def is_ltss(self) -> bool:
+        """determines whether this OS is covered by a different EULA and is not
+        redistributable.
+
```

Comment:
name and description do not really match. it just determines SLES LTSS, it could be non-redistributable for other reasons. 

how about "determines whether this is a SLES version that is currently in LTS support model".